ovs/extensions/services/service.py

```diff
@@ -60,3 +80,54 @@ def __getattr__(cls, item):
             return getattr(ServiceManager.ImplementationClass, item)
 
     __metaclass__ = MetaClass
+
+    @staticmethod
+    def reload():
+        ServiceManager.ImplementationClass = None
+
+    @staticmethod
+    def has_fleet_client():
+        return HAS_FLEET_CLIENT
+
+    @staticmethod
+    def setup_fleet():
```

Comment:
No use for return True or False, because you never use the returned value
Just return in case some check is not passed